app/controllers/pages_controller.rb

```diff
@@ -1,6 +1,7 @@
 class PagesController < ApplicationController
   def index
     solution = Solution.first
+
     IterationsChannel.broadcast!(solution)
   end
-end
+end   
```

Comment:
Trailing white space introduced here.

Reply:
Hmmm. I don't understand why this didn't get picked up by CI. More debugging needed!